tools/msp_merge/msp_merge.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="msp_merge" name="Merge MSP Spectra" version="0.1.0" python_template_version="3.5">
+   <creator>
+        <person
+            givenName="Wudmir"
+            familyName="Rojas"
+            url="https://github.com/wverastegui"
+            identifier="0000-0001-7036-9987" />
+        <person
+            givenName="Helge"
+            familyName="Hecht"
+            url="https://github.com/hechth"
+            identifier="0000-0001-6744-996X" />
+        <organization
+            url="https://www.recetox.muni.cz/"
+            email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+            name="RECETOX MUNI"/>
+    </creator>
+    <description>Merge MSP Spectra files</description>
+    <requirements>
+         <requirement type="package" version="0.14.0">matchms</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+       #set newinput = str("' '").join([str($f) for $f in $input])
+        python3  $__tool_directory__/msp_merge.py
+        --filenames    '$newinput'
+        --outfilename  '$outfile'
+    ]]></command>
+    <inputs>
+        <param name="input" type="data" format="msp" multiple="true"/>
```

Comment:
Planemo test likely fails because `type="data"` with `multiple="true"` attributes disallow to pass collection to this parameter at test time. See [here](https://planemo.readthedocs.io/en/latest/writing_advanced.html#processing-lists-reductions ).

Reply:
Commas have been deleted. ([af48baa](https://github.com/RECETOX/galaxytools/pull/263/commits/af48baa577688acd2b5affa2ce15cd3aec5192eb))